brokers.yml

```diff
@@ -368,23 +375,30 @@ components:
           - single
           - partner
           - married
-          - ...
+          - divorced
+          - widowed
         accomodationType:
           type: string
           description: Type of accomodation of the applicant. The enum values will be provided by the broker so that Moank can receive the raw data.
           enum:
-          - villa
-          - bostadsrätt
-          - hyresrätt
-          - ...
+          - apartment
+          - house
+          - house
```

Comment:
double house here 😉  Probably you ment `rent`? 🤔

Reply:
Updated